Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -22,6 +23,18 @@ public class TapToPlace : MonoBehaviour, IInputClickHandler
         [Tooltip("Supply a friendly name for the anchor as the key name for the WorldAnchorStore.")]
         public string SavedAnchorFriendlyName = "SavedAnchorFriendlyName";
 
+        [Tooltip("Place parent on tap instead of current game object.")]
+        public bool PlaceParentOnTap = false;
+
+        [Tooltip("Specify the parent game object to be moved on tap, if the immediate parent is not desired.")]
+        public GameObject ParentGameObjectToPlace;
+
+        /// <summary>
+        /// Keeps track of if the user is moving the object or not.
+        /// Setting this true will place the object immediately.
```

Comment:
Setting this true will place object immediately makes it sound like the object will not be movable unless the user clicks the object again, but the way it's currently written will actually result in the opposite result.